mimic/rest/fastly_api.py

```diff
@@ -109,6 +110,54 @@ def create_backend(self, request, service_id, version_id):
                                                        service_id, version_id)
         return json.dumps(response)
 
+    @app.route(
+        '/service/<string:service_id>/version/<string:version_id>/condition',
+        methods=['POST'])
+    def create_condition(self, request, service_id, version_id):
+        """
+        Returns Create Condition Response.
```

Comment:
The convention we've been trying to adopt in mimic is to document things like this by linking to relevant API documentation.  Is there a page on Fastly's site you could put a URL to in here, documenting what "create condition response" is, so that future maintainers can easily verify if changes to this method are keeping it consistent with said documentation?